tensilelite/Tensile/KernelWriter.py

```diff
@@ -3057,6 +3081,9 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
       # tail: free G2L Vgpr
       module.add(self.tailLoopFreeVgpr(vgprG2L, moduleMacroG2lVgpr))
 
+      if (kernel["DirectToLdsA"] and kernel["NonDTLTailLoopA"]) or (kernel["DirectToLdsB"] and kernel["NonDTLTailLoopB"]):
```

Comment:
We do not need this if condition.
Checking vgprLW!=-1 should be enought (handled in tailLoopFreeVgpr())